### DISCUSSION TITLE
When I enable the dark theme, the title bar remains white, and the icons appear pixelated.

### DISCUSSION THREAD

**Discussion by ghost** [sumatrapdfreader/sumatrapdf#4579] [Q&A]
![image](https://github.com/user-attachments/assets/eac4d809-b5e8-4fa4-a3df-4ce352b403ab)

**Comment by GitHubRulesOK** [comment]
Hmm still a problem the way windows has no uniform means to theme like it did in XP OR Win 7 (both different then was a big issue)

Pre-Release is perhaps not perfect but better
![image](https://github.com/user-attachments/assets/3ef978d4-7761-4ba3-9c22-c56b3eec5cdd)

**Comment by ilker2495** [comment]
Just chiming in to say I have the same problem and it's kinda driving me nuts.

**Reply by GitHubRulesOK** [reply]
@ilker2495 
The Original Poster was showing version 3.5.2 and all theming enhancements,
including better anti aliased pixel handling (since computer screen rendering IS PIXELS of scaled vector graphic lineart),
can only be seen in 3.6 pre-release as I show above.

Whatever DPI system you have pixels is unavoidable.
